### PR TITLE
feat: sanitise inputs

### LINKED ISSUE #40: Sanitize inputs
Currently, the postgres plugin doesn't sanitize inputs. We should add logic to sanitize our inputs before they're entered into the database. We have some good validation as a starting point in the `user_model.ts` file. We should expand on our validation efforts.

You can test this by following the steps below:

1. Log in with user1@hotmail.com / Userpass1
2. Go to the Settings page.
3. Update your bio with the following:
```
' WHERE id = '1'; UPDATE users SET username = 'test
```

Notice that you're username becomes test.

### PR DESCRIPTION
**Pull request by ebebbington**
Fixes #40 

**Description**

- [x] Deno will now watch our app, restarting when a ts file changes :)
- [x] Improve code and refactor,  to make it scalable and more maintainable for sanitising inputs
- [x] Sanitise inputs inside `BaseModel#query`

### CODE REVIEW

**Comment by ebebbington** [comment]
Tests are HELLA broken and idk why. Gonna revisit another time i think

**Comment by ebebbington** [comment]
18 days have passed... it is time... to show these tests who is boss..

**Comment by ebebbington** [comment]
This PR is now ready for review

**Comment by ebebbington** [comment]
Resolved conflicts, PR is ready for review now

**Comment by ebebbington** [comment]
Had to go back to deno postgres again, because there is a bug with pgc4d with the latest version at compile time, and it seems unmaintained. deno_postgres also has a new maintainer and im confident that will keep on top it of it

**Comment by ebebbington** [comment]
Ok now this PR is done xD

**Review by Guergeiro — APPROVED** [review]
Everything seems fine to me. I just didn't find where the sanitise inputs us done... Is it already handled by postgresql dep?

**Comment by ebebbington** [comment]
Yeah already handled by Postgres, I’ll try get the code that does it

File: src/models/base_model.ts [hunk 1]
```diff
@@ -116,71 +70,68 @@ export default abstract class BaseModel {
...
-      occurrences.splice(occurrences.length - 1);
+  public static async query(
+    query: string,
+    ...args: Array<string | number>
+  ): Promise<
+    { rows: Record<string, unknown>[]; rowCount: number; error?: boolean }
+  > {
+    try {
+      const db = await BaseModel.getDb();
+      const dbResult = args && args.length
```
Comment (ebebbington) [inline src/models/base_model.ts:120]: This bit, when using the second parameter with the Postgres client query method, and $<number> in the query, it sanitises it

**Pull request merged**